spec/features/white_label_language_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+RSpec.describe Unit::WhiteLabelLanguage do
+  before do
+    establish_connection_to_api!
+  end
+
+  after do
+    Unit::WhiteLabelLanguage
+      .find(ENV.fetch('UNIT_LANGUAGE_ID'))
+      .save_config(JSON.parse(fixture('language_initial.json').read))
```

Comment:
Do you think it’s a bit awkward that we’re taking a json string, parsing it into a hash, then turning it back to JSON?

Would it make sense to have a save_json, or perhaps even allow the existing ResourceOperation for Save to support a save(data, raw: true)?